prompt-service/src/unstract/prompt_service/main.py

```diff
@@ -300,8 +297,13 @@ def prompt_processor() -> Any:
                 "Failed due to vector db error",
             )
             return result, 500
+        # Set up llm, embedding and callback manager to collect usage stats
+        # for this context
+        UNCallbackManager.set_callback_manager(
```

Comment:
NIT: We might need other params to track the usage to the prompt studio project. Can be taken up through a separate PR